extension/chrome/elements/compose-modules/formatters/encrypted-mail-msg-formatter.ts

```diff
@@ -175,17 +174,7 @@ export class EncryptedMsgMailFormatter extends BaseMailFormatter {
      *   the recipient will follow transparently. For now, we'll be following the default set
      *   in OpenPGP.js, and later we can make a deliberate choice on how many iterations to use
      *   without having to affect recipient code.
-     *
-     * Another thing to note is that eventually, flowcrypt.com/api web portal functionality will
-     *   be deprecated, and we'll instead run a "shared tenant FES instance" to fill that role.
-     *   Nothing will change for users, but our code on the client will be more streamlined.
-     *   Therefore, eventually, this `if` branch with the line below will be removed once both
-     *   consumers and enterprises use API with the same structure.
      */
-    if (!(await this.view.acctServer.isFesUsed())) {
-      // if flowcrypt.com/api is used
-      newMsg.pwd = await PgpHash.challengeAnswer(newMsg.pwd); // then hash the password to preserve compatibility
-    }
```

Comment:
@tomholub I also removed this `if` branch as per (removed) above comment.
Am I correct?

Reply:
you are correct here